quantum/audio/song_list.h

```diff
@@ -1,4 +1,7 @@
-/* Copyright 2016 Jack Humbert
+/* Any song or sound without a license explicitly states is:
+ *
+ * Copyright 2019 Jack Humbert
```

Comment:
I don't think 2019 would be the correct year to specify for Lines 3-4.

Reply:
Why not?

Reply:
Because they were written prior to this year.

Reply:
```suggestion
 * Copyright 2016 Jack Humbert
```

Reply:
:+1: